apps/docs/introduction.mdx

```diff
@@ -15,7 +15,7 @@ description: 'The new way to build, share, and run code automation bots, aka cod
   alt="Codemod.com hero banner"
 />
 
-At Codemod.com we are building the ecosystem, platform and tooling around code modification bots, aka codemods. The Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
+Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
```

Comment:
Codemod platform helps you create, distribute, and run codemods in codebases of any size.